builtins/amp-img.js

```diff
@@ -122,7 +122,12 @@ export class AmpImg extends BaseElement {
       this.allowImgLoadFallback_ = false;
     }
 
-    this.img_ = new Image();
+    // For inabox SSR, image will have been written directly to DOM so no need
+    // to recreate.  Calling appendChild again will have no effect.
+    if (this.element.hasAttribute('i-amphtml-ssr')) {
+      this.img_ = this.element.querySelector('img');
```

Comment:
Use `scopedQuerySelector` from dom.js - it's pretty important.

Reply:
Done